src/navigation/pathnavigator_lua.inl

```diff
@@ -311,7 +334,9 @@ namespace {
  * Fade rendering to black, jump to the specified node, and then fade in. This is done by
  * triggering another script that handles the logic.
  *
- * If no fade duration is specified, use the property from Navigation Handler
+ * \param navigationState A [NavigationState](#core_navigation_state) to jump to
+ * \param fadeDuration An optional duration for the fading. If not included, the
+ *                     property in Navigation Handler will be used
```

Comment:
Add a reference to the Navigation Handler maybe

Reply:
@ylvaselling, can we link to properties even? :)  How are the URLs for them created?

Reply:
Actually, the Settings part is not in the generated part of the new documentation page. 

@ylvaselling Maybe let's add it as another menu under "Reference"?